File: pythoncollections/my_dictionary/word_count.py
text="hai how are you hai how are you"

#word count hai,2 how,2 are,2 you,2
dict={}
words=text.split(" ")#['hai', 'how', 'are', 'you', 'hai', 'how', 'are', 'you']
print(words)
for word in words:#word=hai,how...
    if(word not in dict):#if hai is not in dict hai,1
        dict[word]=1#dict{hai:1}
    else:
        dict[word]+=1
print(dict)

File: javascript_programs/javascripttypes/wordcount.js
var text = "hello hai hello how";
var word = text.split(" ");
console.log("Text has ", word.length, " words");
console.log(word);
dict={};
for (i = 0; i < word.length; i++) {
    var cnt = 0;
    for (j = 0; j < word.length; j++) {

        // console.log(word[i],word[j])
        if (word[i] == word[j]) {
            cnt = cnt + 1
            if(word in dict)
            {
                console.log("ifffffffffffffffffffffffffff")
                continue;
                
            }
            else{
                 dict[word[i]]=cnt
                }      
            //     }
    
            

        }


    }
    console.log(word[i], "has a count of ", cnt)
}
console.log("word count is \n", dict)